libraries/AP_Follow/AP_Follow.h

```diff
@@ -128,6 +139,7 @@ class AP_Follow
     AP_Int8     _yaw_behave;        // following vehicle's yaw/heading behaviour (see YAW_BEHAVE enum)
     AP_Int8     _alt_type;          // altitude source for follow mode
     AC_P        _p_pos;             // position error P controller
+    AP_Int16    _options;           // options for mount behaviour follow mode
```

Comment:
This doesn't hold up the merge or anything... but I don't see any harm in making all _OPTIONS params 32bit... We should just make it a habit to do that in the future.